cmd/thanos/rule.go

```diff
@@ -92,7 +92,7 @@ func registerRule(m map[string]setupFunc, app *kingpin.Application, name string)
 
 	objStoreConfig := regCommonObjStoreFlags(cmd, "", false)
 
-	queries := cmd.Flag("query", "Addresses of statically configured query API servers (repeatable). The scheme may be prefixed with 'dns+' or 'dnssrv+' to detect query API servers through respective DNS lookups.").
+	queries := cmd.Flag("query", "Addresses of statically configured query API servers (repeatable). The scheme may be prefixed with 'dns+' or 'dnssrv+' to detect query API servers through respective DNS lookups. No scheme prefix allowed (https://github.com/improbable-eng/thanos/issues/1025)").
```

Comment:
I think the comment here doesn't make sense. As we have a sentence `The scheme may be prefixed with 'dns+' or 'dnssrv+'` 
and after that a sentence `No scheme prefix allowed (https://github.com/improbable-eng/thanos/issues/1025)")`

Reply:
yes, We need to reword ;p

It's tricky as we can do `dns+abc:port`, `abc:port` etc, but we cannot put `dns+http://abc:port` or http://abc:port` due to implementation limitations